src/lua-runtime-fastpm.lua

```diff
@@ -33,11 +33,17 @@ function schema.output_redshifts.action(output_redshifts)
     end
 end
 
-schema.declare{name='omega_m',           type='number', required=true, default=0.3 }
+schema.declare{name='omega_m',           type='number', required=true, default=0.3, help='cdm + baryon density parameter at z=0'}
+schema.declare{name='omega_ncdm',           type='number', required=false, default=0.0, help='ncdm density parameter at z=0'}
 schema.declare{name='h',                 type='number', required=true, default=0.7, help="Dimensionless Hubble parameter"}
 schema.declare{name='pm_nc_factor',      type='array:number',  required=true, help="A list of {a, PM resolution}, "}
 schema.declare{name='np_alloc_factor',   type='number', required=true, help="Over allocation factor for load imbalance" }
 schema.declare{name='compute_potential',   type='boolean', required=false, default=false, help="Calculate the gravitional potential."}
+schema.declare{name='n_ncdm',   type='number', required=false, default=1, help="Number of ncdm particles."}
```

Comment:
There is a way to get the number of items in `m_ncdm`, thus you don't need a separate `n_ncdm` parameter. Take a look how this is obtained for the `time_step` parameter.

Reply:
Done 0a7c3f69b489486a7bae72bb2e250039cad51f1b